src/api/app/controllers/webui/comments_controller.rb

```diff
@@ -5,11 +5,13 @@ class Webui::CommentsController < Webui::WebuiController
   def create
     comment = @commented.comments.new(permitted_params)
     User.current.comments << comment
+    @comment = Comment.new
```

Comment:
Why do we have two variables: `comment` and `@comment`? Whats the difference between the two?

Reply:
`@comment = Comment.new` is required inside the partial for the form construction. See: https://github.com/openSUSE/open-build-service/blob/master/src/api/app/views/webui/comment/_new.html.haml#L2

Reply:
Hmm, I think this is why we shouldn't use instance variables in partials... Ok can you please do either:
1. Use a local variable in the _new partial
or
2. Add a comment explaining why we set @comment here